application/libraries/admin/quexmlpdf.php

```diff
@@ -3547,8 +3547,8 @@ protected function drawSingleChoiceVertical($categories, $subquestions = array(a
             }
 
             //draw the response boxes
-            $arraySize = count($subquestions);
-            for ($j = 0; $j < $arraySize; $j++) {
+            $subquestionCount = count($subquestions);
+            for ($j = 0; $j < $subquestionCount; $j++) {
```

Comment:
I don't see what's different? Was `arraySize` redefined somewhere else?

Reply:
Yes. This line was the line redefining it.
This line was the original definition https://github.com/gabrieljenik/LimeSurvey/blob/bc0d8a495e95723a16be805fc3e9a2a89df08c46/application/libraries/admin/quexmlpdf.php#L3494

Reply:
Cool, thank you!